tests/testhelper.py

```diff
@@ -71,6 +72,7 @@ def set_config(settings_dict):
     """Change config-values on the fly, per test"""
 
     def set_config_decorator(func):
+        @wraps(func)
```

Comment:
Why is this needed?

Reply:
As I understand it if its not there then decorators break introspection, which pytest uses to determine which fixtures to pass in.[^1]

```py
@set_config({"dirscan_dir": os.path.join(SAB_CACHE_DIR, "watched")})
def test_adds_valid_nzbs_on_startup(self, fs, mocker, dirscanner):
```

So without it the above sees the wrapped function and not the fixtures I need `fs, mocker, dirscanner`.

[^1]: Effective Python - Item 26